meta(changelog): Update changelog for 8.31.0 (#13734)
getsentry/sentry-javascript: CHANGELOG.md

- "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott

## 8.31.0

### Important Changes

- **feat(node): Add `dataloader` integration (#13664)**

> This release adds a new integration for `dataloader` which is based on `@opentelemetry/instrumentation-dataloader`.

Reviewer: ```suggestion
This release adds a new optional integration for the [`dataloader` package](https://www.npmjs.com/package/dataloader).
```

> This release adds a new optional integration for the [`dataloader` package](https://www.npmjs.com/package/dataloader).

Reviewer 2: It's not optional - it's enabled by default: https://github.com/getsentry/sentry-javascript/blob/dbfd8e2d235a11ee2b59eee13386a0fa85e16495/packages/node/src/integrations/tracing/index.ts#L45

Reviewer: oh thanks

```js
Sentry.init({
  integrations: [Sentry.dataloaderIntegration()],
});
```

### Other Changes

- feat(browser): Add navigation `activationStart` timestamp to pageload span (#13658)
- feat(gatsby): Add optional `deleteSourcemapsAfterUpload` (#13610)
- feat(nextjs): Give app router prefetch requests a `http.server.prefetch` op (#13600)
- feat(nextjs): Improve Next.js serverside span data quality (#13652)

> - feat(node): Add `disableInstrumentationWarnings` option (#13693) feat(nuxt): Adding `experimental_basicServerTracing`

Reviewer 3: ```suggestion
- feat(node): Add `disableInstrumentationWarnings` option (#13693)
- feat(nuxt): Adding `experimental_basicServerTracing` option to Nuxt module (#13643)
```
